ja/core-libraries/httpclient.rst

```diff
@@ -365,9 +365,11 @@ Response オブジェクトは、レスポンスデータを検査するため
 文字列としてレスポンスボディー全体を読み込みます。 ::
 
     // 文字列としてレスポンス全体を読み込み
+    $response->getStringBody();
+    
+    // 3.7.0 以前の方法
```

Comment:
Can you put  the 3.7.0 example in comment on line above `getStringBody`?   Same for other updated examples.